Description: Py3 fix
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2018-02-19

--- openstack-trove-9.0.0~rc1.orig/trove/guestagent/common/operating_system.py
+++ openstack-trove-9.0.0~rc1/trove/guestagent/common/operating_system.py
@@ -173,9 +173,9 @@ def _write_file_as_root(path, data, code
     # out of scope.
     with tempfile.NamedTemporaryFile('w', delete=False) as fp:
         if encode:
-            fp.write(codec.serialize(data))
+            fp.write(codec.serialize(data.encode()))
         else:
-            fp.write(codec.deserialize(data))
+            fp.write(codec.deserialize(data.decode()))
         fp.flush()
         fp.close()  # Release the resource before proceeding.
         copy(fp.name, path, force=True, as_root=True)
