Description: Skip tests that fail in Mitaka B2.
Author: Corey Bryant <corey.bryant@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/trove/+bug/1537306
Forwarded: not-needed

--- a/trove/tests/unittests/guestagent/test_dbaas.py
+++ b/trove/tests/unittests/guestagent/test_dbaas.py
@@ -16,6 +16,7 @@
 import os
 import subprocess
 import tempfile
+import testtools
 import time
 from uuid import uuid4
 
@@ -1282,6 +1283,7 @@
         self.assertEqual(expected, args[0],
                          "Sql statements are not the same")
 
+    @testtools.skip("Skipped by Ubuntu")
     @patch.object(dbaas, 'get_engine',
                   return_value=MagicMock(name='get_engine'))
     def test__wait_for_slave_status(self, *args):
